src/common/options/global.yaml.in

```diff
@@ -163,9 +163,9 @@ options:
   type: str
   level: basic
   desc: list of hosts or addresses to search for a monitor
-  long_desc: This is a comma, whitespace, or semicolon separated list of IP addresses
-    or hostnames. Hostnames are resolved via DNS and all A or AAAA records are included
-    in the search list.
+  long_desc: This is a list of IP addresses or hostnames that are separated by
+  commas, whitespace, or semicolons. Hostnames are resolved via DNS. All A and
```

Comment:
I think the doc build failure here https://jenkins.ceph.com/job/ceph-pr-docs/53322/consoleFull#-85362248744e9240e-b50a-4693-bac0-8a991bac86ac is related to changing the indent.